src/main/java/de/tum/in/www1/artemis/domain/metis/Post.java

```diff
@@ -69,6 +70,30 @@
     @Column(name = "course_wide_context")
     private CourseWideContext courseWideContext;
 
+    @NotNull
+    @Column(name = "is_pinned")
+    private boolean pinned;
+
+    @NotNull
```

Comment:
As `boolean` cannot be `null`, this is not really necessary.

Reply:
will be changed with the introduction of enums anyways....